samples/buildstrategy/buildah/buildstrategy_buildah_cr.yaml

```diff
@@ -17,49 +17,158 @@ spec:
         - |
           set -euo pipefail
 
-          echo "Creating registries config file..."
+          # Parse parameters
```

Comment:
not asking as part of this PR, but I'm curious what would happen if you extracted the bash from this file, put it in a .sh file, and ran `shellcheck` against it

Reply:
I had to do this because it failed as I had too much `"`s at one place. :-) But yeah, I further extended it after this. Would somehow be nice to get the inline script validated.